Rakefile

```
@@ -8,7 +8,13 @@ require "yard/mattr_accessor_handler"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb"]
+  t.test_files = FileList["test/sandbox/**/*_test.rb", "test/view_component/**/*_test.rb"]
+end
+
+Rake::TestTask.new(:engine_test) do |t|
+  t.libs << "test/dummy"
```

```suggestion
  t.libs << "test/engine"
```

👋🏻 I have a slight aversion to using the term `dummy`, might you be willing to change the naming to `engine`?

---

Hm, `engine` is something like keyword for rails I think.
If we want to rename test engine to `engine` - we will have weird naming I think. For eg. test engine will look like:

```
module Engine
  class Engine < ::Rails::Engine
    isolate_namespace Engine
  end
end
```

---

Maybe `test_engine`?

---

Or `example`?